messaging/src/main/java/org/axonframework/deadline/quartz/DeadlineJob.java

```diff
@@ -82,9 +82,9 @@ public class DeadlineJob implements Job {
     public static final String JOB_DATA_SERIALIZER = Serializer.class.getName();
 
     /**
-     * The key under which the {@link SpanFactory} is stored within the {@link SchedulerContext}.
+     * The key under which the {@link DeadlineManagerSpanFactory} is stored within the {@link SchedulerContext}.
      */
-    public static final String SPAN_FACTORY = SpanFactory.class.getName();
+    public static final String SPAN_FACTORY = DeadlineManagerSpanFactory.class.getName();
```

Comment:
Let's make sure this isn't a breaking change!

Reply:
![image](https://github.com/AxonFramework/AxonFramework/assets/3278878/c3cbe58f-373e-4b75-aa41-ec23586d8a92)
It isn't, as it's put in the context during construction of the `QuartzDeadlineManager`

Reply:
:+1:  cool, thanks for checking!